recipes/forest/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "forest" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/informatics-lab/forest/archive/v{{ version }}.tar.gz
+  sha256: d06a17ec394ed737fe265d5c163a4e536be3ae20f10443968ba7a04457b710e0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be `noarch: python`?

Reply:
I'm not sure, does noarch: python mean it's just pure python? In which case, yes it can be

Reply:
Yep that is what that means

Reply:
Great, in that case I'll see if I can add it to the pull request. I'm in an airport at the moment with only a smart phone to develop on but I'll see what I can do